menu/menu.js

```diff
@@ -0,0 +1,76 @@
+const { dialog } = require('electron')
+const { app } = require('electron').app
+var usbList=[];
```

Comment:
What is this for? I can't find a reference to this in the pr besides initialization

Reply:
Line 3 was some leftover code from another project that we used as a starting point. We can remove line 3 from the pull request.